inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -570,24 +557,68 @@ std::vector<PointSpeedPair> InLaneCruisingPlugin::maneuvers_to_points(const std:
 
     ROS_DEBUG_STREAM("Used downtrack: " << starting_downtrack);
 
-    auto lanelets = wm->getLaneletsBetween(starting_downtrack, lane_following_maneuver.end_dist, true);
+    auto lanelets = wm->getLaneletsBetween(starting_downtrack, lane_following_maneuver.end_dist, true, true);
```

Comment:
If this were to be made non-inclusive what is the impact on the complex logic below? Could it simplify anything?

Reply:
I think it's best to keep that logic, as the input `max_starting_downtrack` passed to its parent function has that back_distance buffer, which would inevitably pick up lanechanging lanelets.